api/app/routers/fba.py

```diff
@@ -59,13 +73,89 @@ async def get_zones(run_type: RunType, run_datetime: datetime, for_date: date, _
         return FireZoneAreaListResponse(zones=zones)
 
 
+@router.get('/hfi-thresholds/', response_model=List[HfiThreshold])
+async def get_hfi_thresholds():
```

Comment:
I think this is an unused endpoint, except this function is called in `get_hfi_fuels_data_for_fire_zone` so we can move the function logic to the db/crud layer.